JitsiConference.js

```diff
@@ -1411,6 +1423,35 @@ JitsiConference.prototype.onMemberLeft = function(jid) {
     this._maybeClearSITimeout();
 };
 
+/**
+ * Designates an event indicating that we were kicked from the XMPP MUC.
+ * @param {boolean} isSelfPresence - whether it is for local participant
+ * or another participant.
+ * @param {string} actorJid - the jid of the participant who was initiator
+ * of the kick.
+ * @param {string?} participantJid - when it is not a kick for local participant,
+ * this is the jid of the participant which was kicked.
+ */
+JitsiConference.prototype.onMemberKicked = function(isSelfPresence, actorJid, participantJid) {
+    const actorId = Strophe.getResourceFromJid(actorJid);
```

Comment:
It seems that we are anyway using the resource only, why then we are passing the full jid with the event from ChatRoom.js?

Reply:
Yep, will fix it.